src/main/resources/resources/logisim/strings/analyze/analyze_zh.properties

```diff
@@ -75,7 +75,7 @@ minimizeMaxtermsButton = 优化最大项
 #
 # gui/MinimizeButton.java
 #
-OptimizeLongTimeWarning = 警告，使用此数量的输入优化逻辑函数可能需要很长时间（数小时）！\n您要继续吗？
+OptimizeLongTimeWarning = 警告，使用此数量的输入优化逻辑函数可能需要很长时间（数小时）！\n 您要继续吗？
```

Comment:
In other places, you don't use a space before '!'. In English and many other European languages, there's no space before the exclamation mark '!'. One exception is French.